src/neo/models/actors.js

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash');
+const connection = require('../connection.js');
+const Actor = require('./neo4j/actor.js');
+
+
+const manyActors = (results) => {
+  return results.records.map(r => new Actor(r.get('actor')))
+}
+
+const insert = (session, object) => {
+  return session
+    .run(`CREATE (actor:Actor {
+        id: '${object.id}',
```

Comment:
You can make these expressions generic by doing something like `Object.entries(object).map(([k, v]) => ...).join('\n')` for the values and passing the type as well. That way, you can probably generalize the logic for all the models.